tests/cases/compiler/fakeInfinity3.ts

```diff
@@ -1,20 +1,20 @@
-// @strict: true
-// @declaration: true
-
-export enum Foo {
-    A = 1e999,
-    B = -1e999,
-}
-
-namespace X {
-    type A = 1e999;
-    type B = 2e999;
-
-    export function f(): A {
-        throw new Error()
-    }
-}
-
-export const m = X.f();
-
-export const Infinity = "oops";
+// @strict: true
```

Comment:
What happened to these files? Some sort of line ending bug?

Reply:
I was also wondering about it - but I didn't care as much to investigate this. I first reverted your revert (so that removed those files) and then applied your patch from https://github.com/microsoft/TypeScript/pull/56422/commits/45371af679fe83b5128fe4ee4c4b74b1ad17e7be using `git am` (that brought those files back). So maybe it somehow tripped on this combo - where the same file was both removed and added in the series of commits?

I like the approach that I used since the "commit trace" ends up being quite nice using it. The resulting diff is weird though so maybe it was not worth it. I could try to just squash all of those commits here and it's likely the problem would go away after that. Would you prefer me doing this?

Reply:
I suspect this is just a line endings problem, and these files can just be opened in VS Code and swapped to the other line endings (there's a button at the bottom right for it).

Reply:
You were right - this was the line-ending problem. It's kinda weird that they were changed in this whole process since I literally applied the downloaded patch directly - so I wonder at what stage this was changed... but well, the important thing is that the problem is gone now :p Thanks for the VS Code tip - I didn't know that it can convert all of the existing line endings for me.